gnomad/resources/grch38/gnomad.py

```diff
@@ -1,6 +1,10 @@
 # noqa: D100
 
-from typing import Optional
+import logging
+from typing import Optional, Union
+
+import hail as hl
+from gnomad_qc.v3.create_release.prepare_vcf_data_release import FAF_POPS
```

Comment:
to maintain proper isort order, move imports from "gnomad_qc" below ones from "gnomad"

Reply:
that's odd, I ran isort on it? maybe running black afterwards messed it back up - i'll address it